packages/hint-no-disallowed-headers/src/hint.ts

```diff
@@ -27,13 +27,64 @@ const debug = d(__filename);
  * ------------------------------------------------------------------------------
  */
 
+interface ISpecialHeadersWithMessage {
+    [key: string]: MessageName;
+}
+
 export default class NoDisallowedHeadersHint implements IHint {
 
     public static readonly meta = meta;
 
+    private serverHeaderContainsTooMuchInformation(serverHeaderValue: string): boolean {
```

Comment:
Nit: I'm glad to see this moved out of the constructor, but since it doesn't use any instance state can we either make it `static` or move it outside of the class entirely as a helper function (the latter being my personal preference)?

Reply:
Ah, thanks, I wasn't sure whether to do this or not. But I agree with you, I'd prefer for this to be outside of the class entirely.
I'll address this in this PR soon.